packages/plugins/src/cli/propose/index.ts

```diff
@@ -106,8 +108,15 @@ export const buildParsedConfigArray: BuildParsedConfigArray = async (
       process.exit(1)
     }
 
+    // Check that the system contracts are deployed.
     const provider = new SphinxJsonRpcProvider(rpcUrl)
-    await ensureSphinxAndGnosisSafeDeployed(provider)
+    // This auto-generated wallet is only used for internal testing purposes.
+    const wallet = new ethers.Wallet(getSphinxWalletPrivateKey(0), provider)
+    await ensureSphinxAndGnosisSafeDeployed(
+      provider,
+      wallet,
+      ExecutionMode.Platform
+    )
```

Comment:
This will fail when proposing against a live network since the `fundAccountMaxBalance` function uses the `hardhat_setBalance` rpc method. I actually think we can probably just remove the call to this function entirely since in the propose function we can reasonably assume the contracts are already deployed, but I haven't fully thought that through so use your judgement. 

Please also spend a little bit of time testing the proposal task works on live networks.

Reply:
Nice catch. I removed this call since you're right that it's not necessary. I checked that we can propose with the latest version of this PR on a live network. Also, I'll make a ticket to add an integration test for a live network proposal because this should've been caught by an automated test